consensus/istanbul/backend/test_utils.go

```diff
@@ -134,28 +134,31 @@ func makeBlock(keys []*ecdsa.PrivateKey, chain *core.BlockChain, engine *Backend
 	block, _ = engine.updateBlock(parent.Header(), block)
 
 	// start the sealing procedure
-	results := make(chan *types.Block)
+	results := make(chan *istanbul.BlockConsensusAndProcessResult)
 	go func() {
 		err := engine.Seal(chain, block, results, nil)
 		if err != nil {
 			panic(err)
 		}
 	}()
 
-	// create the sig and call Commit so that the result is pushed to the channel
+	// setup the BlockConsensusAndProcessResult cache
+	cachedResult := &istanbul.BlockProcessResult{}
+	engine.core.CurrentRoundState().SetBlockProcessResult(block.Hash(), cachedResult)
```

Comment:
Out of curiosity, why do you need to set the cached process result here?  Whereas in the other test cases when `Commit(` is called, you don't set the cached process result.

Reply:
It's redundant.